enterprise/libs.amazon/external/jackson-annotations-2.6.0-license.txt

```diff
@@ -0,0 +1,209 @@
+Name: Jackson-annotations
+Origin: FasterXML
+Version: 2.6.0
+License: Apache-2.0
+Description: Core annotations used for value types, used by Jackson data binding package
+URL: http://github.com/FasterXML/jackson
```

Comment:
Wouldn't https://github.com/FasterXML/jackson-annotations be a better url?

Reply:
I think I pulled that URL from the pom file for the library, but agree that your suggestion makes more sense.  I have updated accordingly.

I have also merged Amazon SDK licenses into one file, as requested.